3.6/README.md

```diff
@@ -1,95 +1,90 @@
 # Nominatim Docker (Nominatim version 3.6)
 
-1. Build
-  ```
-  docker build --pull --rm -t nominatim .
-  ```
-  See below for optional build arguments to include postcode data in your image.
 
-2. Copy <your_country>.osm.pbf to a local directory (i.e. /home/me/nominatimdata)
+## Automatic import
 
-3. Initialize Nominatim Database
-  ```
-  docker run -t -v /home/me/nominatimdata:/data nominatim  sh /app/init.sh /data/<your_country>.osm.pbf postgresdata 4
-  ```
-  Where 4 is the number of threads to use during import. In general the import of data in postgres is a very time consuming
-  process that may take hours or days. If you run this process on a multiprocessor system make sure that it makes the best use
-  of it. You can delete the /home/me/nominatimdata/<your_country>.osm.pbf once the import is finished.
+Download the required data, initialize the database and start nominatim in one go
 
+```
+  docker run -it --rm \
+    -e PBF_URL=http://download.geofabrik.de/europe/monaco-latest.osm.pbf \
+    -e REPLICATION_URL=http://download.geofabrik.de/europe/monaco-updates/ \
+    -e IMPORT_WIKIPEDIA=true \
+    -p 5432:5432 \
+    -p 8080:8080 \
+    --name nominatim \
+    stadtnavi/nominatim:3.6
```

Comment:
I think it would be great if we also provide the image here in upstream & use that one. How about using Github Actions to build the image?

Reply:
In our repo we use Github Actions to build and publish the image: https://github.com/stadtnavi/nominatim-docker/blob/master/.github/workflows/docker.yml

I'm more than happy to help you set it up too.

Reply:
The workflow is quite simple. One could also image a step that does an import and runs a query to check that everything is working before building the image.